tests/integration/utils.py

```diff
@@ -29,5 +26,6 @@ def create_file_group(files_qty=1):
 
 # travis-ci.org runs our tests in an overworked virtual machine, which makes
 # timing-related tests unreliable.
-skip_on_travis = unittest.skipIf('TRAVIS' in os.environ,
-                                 'timing tests unreliable on travis')
+skip_on_travis = unittest.skipIf(
+    "TRAVIS" in os.environ, "timing tests unreliable on travis"
+)
```

Comment:
we're not using travis anymore.
is this still a problem in GHA?
if so, rename to `skip_on_ci`

Reply:
didn't notice any problems with tests on CI, removed